.mailmap

```diff
@@ -0,0 +1,21 @@
+Damien Lebrun-Grandie <dalg24@gmail.com>
+Damien Lebrun-Grandie <dalg24@gmail.com> <dalg24+github@gmail.com>
+Damien Lebrun-Grandie <dalg24@gmail.com> <lebrungrandt@ornl.gov>
+Damien Lebrun-Grandie <dalg24@gmail.com> <qdi@ornl.gov>
+Christoph Junghans <junghans@votca.org>
+Christoph Junghans <junghans@votca.org> <junghans@lanl.gov>
+Christoph Junghans <junghans@votca.org> <christoph.junghans@gmail.com>
+Dan Ibanez <daibane@sandia.gov>
+Dan Ibanez <daibane@sandia.gov> <ibaned@users.noreply.github.com>
+Conrad Clevenger <tccleve@sandia.gov>
+Conrad Clevenger <tccleve@sandia.gov> <tccleve@kokkos-dev-2.sandia.gov>
```

Comment:
@tcclevenger Did I get it right, you want to go by Conrad or you want Thomas to appear?